native-auth/native_auth_client.md

```diff
@@ -0,0 +1,17 @@
+## NativeAuthClient
+
+```
+class NativeAuthClient:
+    // config defaults to a NativeAuthClientConfig object with default values
+    constructor(config?: NativeAuthClientConfig);
+
+    // returns an initial token, that is then used for the final token
+    initialize(extra_info?: dict[Any, Any]): string;
```

Comment:
`extra_info` could be part of the NativeAuthClientConfig dto ? Maybe this would be more obvious for the developer when he wants to set the field.
There is a specific reason why this is not part of the dto ?